frontends/election-manager/src/screens/write_ins_screen.tsx

```diff
@@ -188,13 +189,8 @@ export function WriteInsScreen(): JSX.Element {
                   <tr key={contest.id}>
                     <TD nowrap>
                       <Text as="span" muted={!hasWriteIns}>
-                        {contest.section === contest.title ? (
-                          contest.title
-                        ) : (
-                          <React.Fragment>
-                            {contest.section}, <strong>{contest.title}</strong>
-                          </React.Fragment>
-                        )}
+                        {getContestDistrictName(election, contest)},{' '}
+                        <strong>{contest.title}</strong>
```

Comment:
One place where we would normally exclude showing `contest.section` if it matched `contest.title` if they were the same. 

That flexibility was nice but it seems like a poor and opaque pattern, so better to remove it and figure something else out later if we want it.